lib/init.php

```diff
@@ -87,17 +94,19 @@ function beans_load_dependencies() {
 	 * @since 1.0.0
 	 */
 	do_action( 'beans_after_load_api' );
-
 }
 
 add_action( 'beans_init', 'beans_add_theme_support' );
 /**
  * Add theme support.
  *
+ * @since 1.0.0
  * @ignore
+ * @access private
```

Comment:
The access is not private.  Go ahead and remove the `@access` tag.  We do want to ignore it though for the automatic doc generator.